handlers.go

```diff
@@ -218,13 +221,33 @@ func setupGatewayHandler(cfg Config, nd *Node, tracingAuth string) (http.Handler
 			if request.Header.Get("Tracestate") != "" {
 				request.Header.Del("Tracestate")
 			}
+			if request.Header.Get(NoBlockcacheHeader) != "" {
+				request.Header.Del(NoBlockcacheHeader)
+			}
+		}
+
+		// Process cache skipping header
+		if noBlockCache := request.Header.Get(NoBlockcacheHeader); noBlockCache != "" {
+			ds, err := leveldb.NewDatastore("", nil)
```

Comment:
This bring in leveldb as a dependency, but IIRC the levelDB memory datastore behaves a bunch better under parallel block writing/fetching load compared to the mapdatastore + sync datastore wrapper. I don't expect this to be used a ton either way, so if people disagree I'm ok switching to the more basic datastore.